Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/merge_border_vertices.h

```diff
@@ -58,6 +61,82 @@ struct Less_on_point_of_target
   const VertexPointMap& vpm;
 };
 
+template <typename VPM, typename PolygonMesh>
```

Comment:
Add a comment explaining the main idea?

Reply:
There's a comment where the function is called... but I see that the comment is now outdated. I'll update it.